<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Keshav" failures="1" tests="2" name="Error Validation Test" time="27.55" errors="0" timestamp="2024-09-05T15:31:55 IST">
  <testcase classname="seleniumproject.Test.ErrorValidations" name="ProductOrderValidation" time="14.315">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at seleniumproject.Test.ErrorValidations.ProductOrderValidation(ErrorValidations.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- ProductOrderValidation -->
  <testcase classname="seleniumproject.Test.ErrorValidations" name="submitOrder" time="0.533"/>
</testsuite> <!-- Error Validation Test -->
